homeassistant/components/light/tplink.py

```diff
@@ -5,14 +5,19 @@
 https://home-assistant.io/components/light.tplink/
 """
 import logging
+import colorsys
 from homeassistant.const import (CONF_HOST, CONF_NAME)
 from homeassistant.components.light import (
-    Light, ATTR_BRIGHTNESS, ATTR_COLOR_TEMP, ATTR_KELVIN,
-    SUPPORT_BRIGHTNESS, SUPPORT_COLOR_TEMP)
+    Light, ATTR_BRIGHTNESS, ATTR_COLOR_TEMP, ATTR_KELVIN, ATTR_RGB_COLOR,
+    SUPPORT_BRIGHTNESS, SUPPORT_COLOR_TEMP, SUPPORT_RGB_COLOR)
 from homeassistant.util.color import \
     color_temperature_mired_to_kelvin as mired_to_kelvin
-from homeassistant.util.color import \
-    color_temperature_kelvin_to_mired as kelvin_to_mired
+from homeassistant.util.color import (
+    color_temperature_kelvin_to_mired as kelvin_to_mired)
+
+from typing import Tuple, TYPE_CHECKING
+if TYPE_CHECKING:
+    from pyHS100 import SmartBulb
```

Comment:
Fixed by not using '# type: SmartBulb' but adding the type hint on parameter list of the method.